docs/dietpi_tools.md

```diff
@@ -189,7 +158,8 @@ When you execute the certificate installation it also installs it for your selec
 
 ### DietPi VPN
 
-DietPi-VPN is a combination of OpenVPN installation and DietPi front end GUI. Allowing all VPN users to quickly and easily connect to any NordVPN, ProtonVPN, or any other server that uses OpenVPN in TCP or UDP, using only open source software.
+DietPi-VPN is a combination of OpenVPN installation and DietPi front end GUI. Allowing all VPN users to quickly and easily connect to any NordVPN, ProtonVPN, or any other server that uses OpenVPN in TCP or UDP, using only open source software.  
+Run `dietpi-vpn`.
```

Comment:
````suggestion
DietPi-VPN is a combination of OpenVPN installation and DietPi front end GUI. Allowing all VPN users to quickly and easily connect to any NordVPN, ProtonVPN, or any other server that uses OpenVPN in TCP or UDP, using only open source software. To start DietPi-VPN, use the following command:

```sh
dietpi-vpn
```
````
Or is a code block too large at this place? Same change, in case, can be done to DietPi-DDNS below as well.